### ISSUE TITLE
Is the project dead?

### ISSUE THREAD

**Issue by alexandru0-dev** [bytefury/crater-mobile#64]

**Comment by kenjibailly** [comment]
I would love to know as well, can't install the app from the play store as it's made for an older version of Android.

**Comment by alexandru0-dev** [comment]
Yup, on crater server repo u can read about them stopping development 3 months ago and it's obvious that also this mobile app is going to be abandoned.

**Comment by kenjibailly** [comment]
> Yup, on crater server repo u can read about them stopping development 3 months ago and it's obvious that also this mobile app is going to be abandoned.

That's a shame, it was exactly what I needed. A platform that could create invoices and customers using an API and good documentation.
Did you find any alternatives?
